src/BPEssentials/CooldownHandler.cs

```diff
@@ -0,0 +1,106 @@
+﻿using BrokeProtocol.Collections;
+using Newtonsoft.Json;
+using System.Collections;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Net.Http;
+using System.Threading.Tasks;
+using UnityEngine;
+
+namespace BPEssentials.Cooldowns
+{
+    public class CooldownHandler
+    {
+        public Dictionary<ulong, Dictionary<string, Dictionary<string, int>>> Cooldowns { get; set; } = new Dictionary<ulong, Dictionary<string, Dictionary<string, int>>>();
+
+        private bool ready;
+
+        private readonly string CustomDataKey = "bpe:cooldowns";
```

Comment:
Suggested to change to a getter only property.
```suggestion
        public string CustomDataKey { get; } = "bpe:cooldowns";
```